jans-cedarling/cedarling/src/bootstrap_config/decode.rs

```diff
@@ -0,0 +1,310 @@
+use super::{
```

Comment:
We added three more bootstrap properties:
* **`CEDARLING_USER_AUTHZ`** : When `enabled`, Cedar engine authorization is queried for a User principal.

* **`CEDARLING_WORKLOAD_AUTHZ`** : When `enabled`, Cedar engine authorization is queried for a Workload principal.

* **`CEDARLING_USER_WORKLOAD_BOOLEAN_OPERATION`** :  `AND`, `OR`

Not sure if you want to just add these now, even though they're not implemented.

Reply:
oh okay, i'll just add those later so we don't have to re-implement.